includes/class-wc-stripe-account.php

```
@@ -447,7 +453,35 @@ public function get_existing_webhook() {
+		}
+
+		$was_oc_enabled = 'yes' === ( $old_value['optimized_checkout_element'] ?? '' );
+		$was_ap_enabled = 'yes' === ( $old_value['adaptive_pricing'] ?? '' );
+
+		// If Adaptive Pricing and Optimized Checkout are both unchanged in the new value, do nothing.
+		if ( $was_ap_enabled === $is_ap_enabled && $was_oc_enabled === $is_oc_enabled ) {
+			return;
+		}
+
+		$this->maybe_reconfigure_webhooks_on_update();
+	}
```

PHPMD (2.15.0)

[error] The method maybe_reconfigure_webhooks_after_adaptive_pricing_enabled is not named in camelCase. (undefined)

(CamelCaseMethodName)